paperqa/readers.py

```
@@ -168,6 +168,18 @@ def parse_json(
             Text(text=t, name=f"{docname}", doc=doc, page_text = page_text, is_table = is_table)
             for i, t in enumerate(raw_texts)
         ]
+    elif "is_faq" in json_contents:
+        docname = json_contents['filename']
+
+        for faq_entry in json_contents['faqs']:
+            text = faq_entry.get('text', '')
```

Don't give default argument to dict get. let it fail for the doc and raise a exception.